docs/source/getting-started/tabular-data/computing-basic-statistics.ipynb

```diff
@@ -0,0 +1,472 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Computing Basic Statistics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this section, you will learn to compute essential statistical measures such as: \n",
+    "\n",
+    "* Sum \n",
+    "* Mean \n",
+    "* Median\n",
+    "* Quantiles\n",
+    "\n",
+    "For each method, we will compare the actual values to the differentially private values to demonstrate utility. The [documentation](../../api/python/opendp.polars.html) also provides more information about the methods. We will use the [sample data](https://github.com/opendp/dp-test-datasets/blob/master/data/sample_FR_LFS.csv.zip) from the Labour Force Survey in France. "
```

Comment:
- Maybe, change the first sentence to, "compare the values of the true measures to those of the differentially private ones".
- The second sentence to, "these methods".